Assets/Students/HenryLiao/SpinsOnY.cs

```diff
@@ -1,14 +0,0 @@
-﻿using System.Collections;
-using System.Collections.Generic;
-using UnityEngine;
-
-public class SpinsOnY : MonoBehaviour
```

Comment:
Make this a generic spin class.
Public Vector3 variable to set the spin speed on each axis.